my_game.py

```diff
@@ -387,7 +390,10 @@ def on_update(self, delta_time):
         self.asteroid_list.update()
 
         # Temporary Asteroid test
-        new_asteroid = Asteroid()
+        new_asteroid = Asteroid(
+            center_x=SCREEN_WIDTH//2,
+            center_y=SCREEN_HEIGHT//2
+        )
         self.asteroid_list.append(new_asteroid)
```

Comment:
This is a bug from a previous commit. Please fix it. We don't want to add an Asteroid on every frame :scream:

Reply:
Alright!

Reply:
I fixed it. The problem was that it was always updating to be the same as change_x and change_y which both is 0.

Reply:
Good catch!